packages/client/src/pages/quiz-session/ui/QuizBox.tsx

```diff
@@ -0,0 +1,159 @@
+import { Dispatch, SetStateAction, useState, useRef, useEffect, useCallback } from 'react';
+
+import { getQuizSocket } from '@/shared/utils/socket';
+
+interface QuizBoxProps {
+  reactionStats: {
+    easy: number;
+    hard: number;
+  };
+  setReactionStats: Dispatch<SetStateAction<{ easy: number; hard: number }>>;
```

Comment:
``` ts
  interface Reaction {
    easy: number;
    hard: number;
 }
```
이런식으로 중첩된 부분을 제거할 수 있을 거 같아요!

```suggestion
  setReactionStats: Dispatch<SetStateAction<Reaction>>;
```

Reply:
아래에 ReactionData가 있으니 그거 사용하면 되겠네요!